src/test/scala/chiselTests/Direction.scala

```diff
@@ -117,15 +117,15 @@ class DirectionSpec extends ChiselPropSpec with Matchers with Utils {
     })
   }
 
-  property("Explicitly directioned but empty Bundles should cause direction errors") {
-    an[Exception] should be thrownBy extractCause[Exception] {
-      ChiselStage.elaborate(new Module {
-        val io = IO(new Bundle {
-          val foo = UInt(8.W)
-          val x = Input(new Bundle {})
-        })
+  property(
+    "Explicitly directioned but empty Bundles should not cause direction errors because Chisel and chisel3 directionality are merged"
+  ) {
+    ChiselStage.elaborate(new Module {
+      val io = IO(new Bundle {
+        val foo = UInt(8.W)
+        val x = Input(new Bundle {})
       })
-    }
+    })
   }
```

Comment:
Can you add a couple of tests showing the mixed semantics? For example, is the following now legal?
```scala
val io = IO(new Bundle {
  val a = Input(UInt(8.W))
  val b = Output(UInt(8.W))
  val c = UInt(8.W)
  val d = Flipped(UInt(8.W))
})
```